CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## Unreleased
 
+## [0.19.2] - 2023-10-10
```

Comment:
We normally only add this line when we cut a release, and leave all the current stuff under `Unreleased`